client/components/payments-activity/survey/style.scss

```diff
@@ -0,0 +1,59 @@
+.wcpay-payments-activity__survey {
+	$inner-container-max-width: 513px;
```

Comment:
Looks like an odd number, can it be done with flex filling the parent's width?

Reply:
I've removed redundant max-width styling in 824c8d976911795f9354a2fb1b8cbb2e768c0be9

Without a max-width for the form container, filling the width looks awkward and deviates too far from the design IMO.

![image](https://github.com/Automattic/woocommerce-payments/assets/3147296/2c703680-209a-4f61-aeae-af7bbbc14580)

So I've kept the max-width but rounded it to `500px`.